README.rst

```diff
@@ -208,9 +208,9 @@ To upload a file, pass either a file-like object or a tuple of
     >>> data = StringIO('StringIO instance has no .name attribute!')
     >>> from smartfile import BasicClient
     >>> api = BasicClient()
-    >>> api.post('/path/data/', file=('foobar.png', data))
-    >>> # Or use a file-like object with a name attribute
-    >>> api.post('/path/data/', file=file('foobar.png', 'rb'))
+    >>> f = ('foobar.png', data)
+    >>> api.upload(f)
+
```

Comment:
Yeah everything was actually wrong. Newest version fixes this issue so that StringIO is no longer required.